pkg/syslutil/cmd_config.go

```diff
@@ -12,8 +13,12 @@ func ReadCMDFlags(configPath string) ([]string, error) {
 	if ferr != nil {
 		return flags, ferr
 	}
-	str := string(data)
-	flags = strings.Fields(strings.TrimSpace(str))
+	re := regexp.MustCompile("(\\S+\"[^\"]+\")|\\S+")
+	flags = re.FindAllString(strings.TrimSpace(string(data)), -1)
+
+	for i, flag := range flags {
+		flags[i] = strings.ReplaceAll(flag, "\"", "")
```

Comment:
This is dangerous. Sometimes you want to pass a double-quote as part of a parameter. Better to use a regex to match a common flag-or-arg pattern: `^(-+[^=]+=)?"?(.*?)"?$`.

Reply:
Checking

Reply:
Add a new pattern to parse flags config: `(-+[^=\s\n]+)?((=?"[^=\n]+")|(=?[^=\s\n]+))?\s*`